From be26bcf588c1dff9f8a867f764bb46b0d6389081 Mon Sep 17 00:00:00 2001
From: Chris Liddell <chris.liddell@artifex.com>
Date: Fri, 23 Jan 2015 09:48:50 +0000
Subject: [PATCH] Update URW fonts to Jan 2015 release.

The families NimbusMono, NimbusRoman and NimbusSans have been augmented with
production quality Greek and Cyrillic glyphs. These correspond to our
substitutes for the PDF base 14 fonts

 gs/Resource/Init/Fontmap.GS               |  80 +++++++++++++++---------------
diff --git a/gs/Resource/Init/Fontmap.GS b/gs/Resource/Init/Fontmap.GS
index 313399b..b8ef6f1 100644
--- a/gs/Resource/Init/Fontmap.GS
+++ b/gs/Resource/Init/Fontmap.GS
@@ -81,49 +81,49 @@
 
 % Aliases
 
-/Bookman-Demi			/URWBookmanL-DemiBold	;
-/Bookman-DemiItalic		/URWBookmanL-DemiBoldItal	;
-/Bookman-Light			/URWBookmanL-Ligh	;
-/Bookman-LightItalic		/URWBookmanL-LighItal	;
-
-/Courier			/NimbusMonL-Regu	;
-/Courier-Oblique		/NimbusMonL-ReguObli	;
-/Courier-Bold			/NimbusMonL-Bold	;
-/Courier-BoldOblique		/NimbusMonL-BoldObli	;
-
-/AvantGarde-Book		/URWGothicL-Book	;
-/AvantGarde-BookOblique		/URWGothicL-BookObli	;
-/AvantGarde-Demi		/URWGothicL-Demi	;
-/AvantGarde-DemiOblique		/URWGothicL-DemiObli	;
-
-/Helvetica			/NimbusSanL-Regu	;
-/Helvetica-Oblique		/NimbusSanL-ReguItal	;
-/Helvetica-Bold			/NimbusSanL-Bold	;
-/Helvetica-BoldOblique		/NimbusSanL-BoldItal	;
-
-/Helvetica-Narrow		/NimbusSanL-ReguCond	;
-/Helvetica-Narrow-Oblique	/NimbusSanL-ReguCondItal	;
-/Helvetica-Narrow-Bold		/NimbusSanL-BoldCond	;
-/Helvetica-Narrow-BoldOblique	/NimbusSanL-BoldCondItal	;
-
-/Palatino-Roman			/URWPalladioL-Roma	;
-/Palatino-Italic		/URWPalladioL-Ital	;
-/Palatino-Bold			/URWPalladioL-Bold	;
-/Palatino-BoldItalic		/URWPalladioL-BoldItal	;
-
-/NewCenturySchlbk-Roman		/CenturySchL-Roma	;
-/NewCenturySchlbk-Italic	/CenturySchL-Ital	;
-/NewCenturySchlbk-Bold		/CenturySchL-Bold	;
-/NewCenturySchlbk-BoldItalic	/CenturySchL-BoldItal	;
-
-/Times-Roman			/NimbusRomNo9L-Regu	;
-/Times-Italic			/NimbusRomNo9L-ReguItal	;
-/Times-Bold			/NimbusRomNo9L-Medi	;
-/Times-BoldItalic		/NimbusRomNo9L-MediItal	;
+/Bookman-Demi			/BookmanURW-DemBol	;
+/Bookman-DemiItalic		/BookmanURW-DemBolIta	;
+/Bookman-Light			/BookmanURW-Lig	;
+/Bookman-LightItalic		/BookmanURW-LigIta	;
+
+/Courier			/NimbusMon-Reg	;
+/Courier-Oblique		/NimbusMon-Obl	;
+/Courier-Bold			/NimbusMon-Bol	;
+/Courier-BoldOblique		/NimbusMon-BolObl	;
+
+/AvantGarde-Book		/URWGothic-Boo	;
+/AvantGarde-BookOblique		/URWGothic-BooObl	;
+/AvantGarde-Demi		/URWGothic-Dem	;
+/AvantGarde-DemiOblique		/URWGothic-DemObl	;
+
+/Helvetica			/NimbusSan-Reg	;
+/Helvetica-Oblique		/NimbusSan-Ita	;
+/Helvetica-Bold			/NimbusSan-Bol	;
+/Helvetica-BoldOblique		/NimbusSan-BolIta	;
+
+/Helvetica-Narrow		/NimbusSanNar-Reg	;
+/Helvetica-Narrow-Oblique	/NimbusSanNar-Ita	;
+/Helvetica-Narrow-Bold		/NimbusSanNar-Bol	;
+/Helvetica-Narrow-BoldOblique	/NimbusSanNar-BolIta	;
+
+/Palatino-Roman			/PalladioURW-Rom	;
+/Palatino-Italic		/PalladioURW-Ita	;
+/Palatino-Bold			/PalladioURW-Bol	;
+/Palatino-BoldItalic		/PalladioURW-BolIta	;
+
+/NewCenturySchlbk-Roman		/CenturySchURW-Rom	;
+/NewCenturySchlbk-Italic	/CenturySchURW-Ita	;
+/NewCenturySchlbk-Bold		/CenturySchURW-Bol	;
+/NewCenturySchlbk-BoldItalic	/CenturySchURW-BolIta	;
+
+/Times-Roman			/NimbusRom-Reg	;
+/Times-Italic			/NimbusRom-Ita	;
+/Times-Bold			/NimbusRom-Med	;
+/Times-BoldItalic		/NimbusRom-MedIta	;
 
 /Symbol				/StandardSymL	;
 
-/ZapfChancery-MediumItalic	/URWChanceryL-MediItal	;
+/ZapfChancery-MediumItalic	/ChanceryURW-MedIta	;
 
 /ZapfDingbats			/Dingbats	;
 
-- 
1.9.1
